lxd/api_cluster.go

```diff
@@ -76,6 +76,23 @@ var clusterCertificateCmd = APIEndpoint{
 	Put: APIEndpointAction{Handler: clusterCertificatePut},
 }
 
+var clusterGroupsCmd = APIEndpoint{
+	Path: "cluster/groups",
+
+	Get:  APIEndpointAction{Handler: clusterGroupsGet, AccessHandler: allowAuthenticated},
+	Post: APIEndpointAction{Handler: clusterGroupsPost, AccessHandler: allowAuthenticated},
+}
+
+var clusterGroupCmd = APIEndpoint{
+	Path: "cluster/groups/{name}",
+
+	Get:    APIEndpointAction{Handler: clusterGroupGet, AccessHandler: allowAuthenticated},
+	Post:   APIEndpointAction{Handler: clusterGroupPost, AccessHandler: allowAuthenticated},
+	Put:    APIEndpointAction{Handler: clusterGroupPut, AccessHandler: allowAuthenticated},
+	Patch:  APIEndpointAction{Handler: clusterGroupPatch, AccessHandler: allowAuthenticated},
+	Delete: APIEndpointAction{Handler: clusterGroupDelete, AccessHandler: allowAuthenticated},
+}
```

Comment:
That looks quite wrong, only the `Get` operations should use `allowAuthenticated`, otherwise all non-admin users will be able to alter the groups.